docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -8,68 +8,42 @@ for examples using the
       seed funders.
    - **Treasury tokens** that are owned by the SNS governance canister which can be
       spent by the SNS community according to their needs.
-   - **Swap tokens** which are owned by the SNS and swapped for ICP tokens.
-      Initially, parts of the SNS swap tokens are swapped in exchange for ICP tokens
-      in an initial decentralization swap. If the swap is successful, the participants
-      will receive SNS tokens in a basket of neurons. 
-      If not all of the swap tokens are sold in the initial swap, the rest of the
-      swap tokens are reserved for future swaps (separate from the treasury). 
-      Note that future swap are not yet designed, but having the tokens reserved
-      makes SNSs forward compatible to such a feature if it is added in the future.
-
-<!-- NEW BULLET POINT:
-**Decentralization swap parameters**: this defines the parameters of the decentralization swap. MAKE SUB-BULLETS
-**Rules of the swap** such as the minimum number of ICP tokens that the swap must collect to be successful and the maximum number of ICP tokens that it will collect. 
-**Conditions of swap participation** such as the definition of a text that must be confirmed by users when participating in the swap and the possibility to define countries from which users cannot participate (by IP addresses).
-**Fallback controllers of the dapp:** Another important parameter is the set of fallback controllers of the dapp.
-   If a decentralization swap fails, for example because the targeted minimum number
-   of ICP tokens could not be collected, the control of the dapp canister(s) is
-   handed back to the principals defined in these parameters.
-   Normally, this will be set to principals owned by the developers that controlled the dapp
-   before the attempt to decentralize it. 
--->
+   - **Swap tokens** which are owned by the SNS swapped in exchange for ICP tokens
+      in the decentralization swap. If the swap is successful, the participants
+      will receive SNS tokens in a basket of neurons.
+- **Decentralization swap parameters**: this defines the parameters of the decentralization swap: 
+  - **Rules of the swap** such as the minimum number of ICP tokens that the swap must collect to be successful and the maximum number of ICP tokens that it will collect. 
+  - **Conditions of swap participation** such as the definition of a text that must be confirmed by users when participating in the swap and the possibility to define countries from which users cannot participate (by IP addresses).
+  
+- **Fallback controllers of the dapp**: If a decentralization swap fails, for example because the targeted minimum number of ICP tokens could not be collected, the control of the dapp canister(s) is handed back to the principals defined in these parameters. Normally, this will be set to principals owned by the developers that controlled the dapp before the attempt to decentralize it.
```

Comment:
```suggestion
- **Fallback controllers of the dapp**: if a decentralization swap fails, for example because the targeted minimum number of ICP tokens could not be collected, the control of the dapp canister(s) is handed back to the principals defined in these parameters. Normally, this will be set to principals owned by the developers that controlled the dapp before the attempt to decentralize it.
```